tutorials/toolkits/mplot3d.py

```diff
@@ -14,24 +14,21 @@
 
 Getting started
 ---------------
-An Axes3D object is created just like any other axes using
-the projection='3d' keyword.
-Create a new :class:`matplotlib.figure.Figure` and
-add a new axes to it of type :class:`~mpl_toolkits.mplot3d.Axes3D`::
+3D Axes (of class `Axes3D`) are created by passing the ``projection="3d"``
```

Comment:
For some reason, all the single-backtick links added here to the tutorial do not work:
https://6188-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/tutorials/toolkits/mplot3d.html#sphx-glr-tutorials-toolkits-mplot3d-py

Reply:
That was already the case before: https://matplotlib.org/tutorials/toolkits/mplot3d.html.